content/updates/2021-02/5-Up-to-10-millions-assets.md

```diff
@@ -0,0 +1,14 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Up to 10 millions of assets per asset family
+::: meta-data type="New" features="Scalability, Governance" available="mid-February" 
+
+It's now possible to have up to 10 million assets per asset family. Indeed, with time, your catalog grows as you add more and more products. As a result, your number of pictures and files can also double or triple.
```

Comment:
```suggestion
In the era of Product Experience, media files like images, videos, product sheets are more than ever needed to deliver compelling and personalized experiences to your customers. So as your media library grows, Akeneo needs to comply with your needs, that is why you can now have up to 10 million assets per asset family. Make sure your storage is well-sized! 
```

Reply:
not sure about the storage part.. ;)

Reply:
Yes, not sure either. You can remove the last sentence.

Reply:
removed.